param/low_level_motion_0.05.yaml

```diff
@@ -40,3 +40,5 @@ roll_max_rate     : 1.0
 yaw_max           : 3.0
 yaw_min           : -3.0
 yaw_max_rate      : 600.0
+
+update_frequency: 60.0
```

Comment:
I see. There's not a big issue leaving as is. However I'd like to entertain ideas to remove this variable entirely.